src/module/add.js

```diff
@@ -96,4 +108,26 @@ function added() {
   }
 }
 
-export { tasks, desc, added };
\ No newline at end of file
+function removeall() {
+  const checkboxes = document.querySelectorAll('.box:checked');
+  for (let i = 0; i < checkboxes.length; i++) {
+    const checkbox = checkboxes[i];
+    const parent = checkbox.parentElement.parentElement;
+    const index = tasks.findIndex((task) => task.index === parseInt(parent.dataset.index));
+    tasks.splice(i, 1);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    parent.remove();
+  }
+  for (let i = 0; i < tasks.length; i++) {
+    tasks[i].index = i;
+  }
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+
```

Comment:
- Good job implementing the clear all functionality successfully :clap: , however, as per the project requirement [here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=function%20for%20the%20%22Clear%20all%20completed%22%20button%20(use%20filter()%20method) kindly use the filter method to get rid of the completed tasks :recycle: